src/pages/listing/index.js

```diff
@@ -210,6 +210,9 @@ export function controller() {
 
         if(window.confirm("Remove " + data.name + "?")) {
             ref.remove().catch(console.error.bind(console));
+
+            // Can't figure out why `ref.remove()` was redirecting to "/"
```

Comment:
https://github.com/tivac/crucible/blob/master/src/pages/content-edit/index.js#L55-L57 ? 

Remember that `firebase.on` will fire any time any updates occur to the item, which could definitely include deletion.